docs/docker_deployment_guide.md

```diff
@@ -306,9 +306,16 @@ docker compose exec redis redis-cli
 
 ## Backup and restore
 
+Backup:
+
+```bash
+docker compose exec -it db pg_dump -U kbin kbin > dump.sql
```

Comment:
is kbin still supposed to be here?

Reply:
Yes for backwards compatibility the default username and database name are kbin. Users from kbin still have the username/db name kbin.